data/test/vtgate/dml_cases.txt

```diff
@@ -187,6 +187,104 @@
   }
 }
 
+# insert unsharded with select
+"insert into main1 select id from main2"
+{
+  "Original": "insert into main1 select id from main2",
+  "Instructions": {
+    "Opcode": "InsertUnsharded",
+    "Keyspace": {
+      "Name": "main",
+      "Sharded": false
+    },
+    "Query": "insert into main1 select id from main2",
+    "Table": "main1"
+  }
+}
+
+# insert unsharded, invalid value for auto-inc
+"insert into main2(id, val) values(1.0, 'aa')"
+"could not convert val: 1.0, pos: 0: strconv.ParseUint: parsing "1.0": invalid syntax"
+
+# insert unsharded, column present
+"insert into main2(id, val) values(1, 'aa')"
+{
+  "Original": "insert into main2(id, val) values(1, 'aa')",
+  "Instructions": {
+    "Opcode": "InsertUnsharded",
+    "Keyspace": {
+      "Name": "main",
+      "Sharded": false
+    },
+    "Query": "insert into main2(id, val) values (:__seq0, 'aa')",
+    "Table": "main2",
+    "Generate": {
+      "Opcode": "SelectUnsharded",
+      "Keyspace": {
+        "Name": "main",
+        "Sharded": false
+      },
+      "Query": "select next :n values from `seq`",
+      "Value": [
+        1
+      ]
+    }
+  }
+}
+
+# insert unsharded, column absent
+"insert into main2(val) values('aa')"
+{
+  "Original": "insert into main2(val) values('aa')",
+  "Instructions": {
+    "Opcode": "InsertUnsharded",
+    "Keyspace": {
+      "Name": "main",
+      "Sharded": false
+    },
+    "Query": "insert into main2(val, id) values ('aa', :__seq0)",
+    "Table": "main2",
+    "Generate": {
+      "Opcode": "SelectUnsharded",
+      "Keyspace": {
+        "Name": "main",
+        "Sharded": false
+      },
+      "Query": "select next :n values from `seq`",
+      "Value": [
+        null
+      ]
+    }
+  }
+}
+
+# insert unsharded, multi-val
+"insert into main2(id, val) values(1, 'aa'), (null, 'bb')"
```

Comment:
just checking are we supporting these kind of queries in which auto-inc value is supplied in one tuple but needs to be generated for other tuple.
I think we discussed we are going to fail these type of queries.